tests/regression/36-apron/91-mine14-5b-no-threshhold.c

```diff
@@ -46,7 +46,9 @@ int main(void) {
   pthread_create(&id, NULL, t_fun, NULL);
   pthread_create(&id2, NULL, t_fun2, NULL);
   pthread_mutex_lock(&mutex);
-  __goblint_check(x==y); //TODO
+  __goblint_check(x==y);
```

Comment:
Why does this now pass all of a sudden?

Reply:
Is there any reason this shouldn't have worked all along? `x == y` has nothing to do with widening thresholds. d0364dbbdc8dcb3ace1755729b696ec9ecaaa626 introduces the additional bounds checks where thresholds matter at all.